.github/workflows/python-package-develop.yml

```diff
@@ -38,15 +61,19 @@ jobs:
           - python-version: 3.8
             api-key: STAGING_LABELBOX_API_KEY_2
             da-test-key: DA_GCP_LABELBOX_API_KEY
+            sdk-version: ${{ fromJson(needs.get_sdk_versions.outputs.sdk_versions)[3] }}
```

Comment:
Do we not want to test each version against each python version? This tests 1 version against 1 python version

Reply:
So this is a good question. If we do 5 sdk versions vs 5 python versions that is 25 pipelines. I do not think we can run it on stage.
In my experience, if we have issues for older sdk versions that would be due to api changes
I wanted to run this test for every PR to detect early if there are any backward-compatibility issues in the api